package-lock.json

```diff
@@ -79,11 +79,12 @@
       "integrity": "sha512-xdA65Z/U7++Y7L9Uwh8Q8OVOs6qgFz+fb7GAzHFjpr1icO37B//xdzLXm7ZRgA19RWrsNe1nme3h896igJSvvw=="
```

Comment:
I think if package-lock.json is updated, I'd expect to see changes to package.json as well.